src/common/math_utils.hpp

```diff
@@ -310,6 +310,13 @@ inline U mish_fwd(T s) {
     return (U)(s * ::tanhf(v));
 }
 
+template <typename T,
+        typename U = typename utils::remove_reference<T>::type>
+inline U hsigmoid_fwd(T s) {
+    float v = s + 3.0f;
+    return (U)(v < 0.0f ? 0.0f : v > 6.0f ? 1.0f : v / 6.0f);
```

Comment:
Maybe I missed smt by according to [the spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/HSigmoid_5.md) this formula is not correct. Should be similar to `hswish_fwd` but w/o final multiplication on src:
```suggestion
float v = s + 3.0f;
v = v > 0.0f ? v : 0.0f;
v = v < 6.0f ? v : 6.0f;
return (U)(v / 6.0f);
```

Please correct me if I am wrong.

Reply:
It is the same. But I rewrote this function line by line like your sample. Thank you

Reply:
Ok, I see. Anyway multi line implementation looks much more readable from my perspective.